plugins/GSdx/GSDevice11.cpp

```diff
@@ -152,6 +151,23 @@ bool GSDevice11::Create(const std::shared_ptr<GSWnd> &wnd)
 		return false;
 	}
 
+	{	// HACK: check nVIDIA
+		bool is_vendor_id_nvidia = false;
+		IDXGIDevice *dxd;
+		IDXGIAdapter *dxa;
+		DXGI_ADAPTER_DESC dxad;
+
+		if(SUCCEEDED(m_dev->QueryInterface(IID_PPV_ARGS(&dxd))) && SUCCEEDED(dxd->GetParent(IID_PPV_ARGS(&dxa))) && SUCCEEDED(dxa->GetDesc(&dxad)))
+		{
+			is_vendor_id_nvidia = dxad.VendorId == 0x10DE;
+		}
+
+		bool is_native = theApp.GetConfigI("upscale_multiplier") == 1;
+		int is_spritehack = theApp.GetConfigB("UserHacks") ? theApp.GetConfigI("UserHacks_SpriteHack") : 0;
```

Comment:
I made the suggestion to change `isNative` to `is_native` because `isNative` uses camel case whereas `is_native` uses snake case. I did not suggest to add `is_` to all the variables. I'm not sure `is_spritehack` makes that much sense.

Personally I would have written `!nvidia_gpu || native_resolution || spritehack_enabled` because it's more readable to me than `(!is_vendor_id_nvidia || is_spritehack || is_native)`and would have also changed `spritehack_enabled` to a bool and changed the expression to `theApp.GetConfigB("UserHacks") && theApp.GetConfigI("UserHacks_SpriteHack")`.